python/paddle/nn/functional/__init__.py

```diff
@@ -83,6 +83,7 @@
 from .loss import margin_ranking_loss  # noqa: F401
 from .loss import mse_loss  # noqa: F401
 from .loss import nll_loss  # noqa: F401
+from .loss import poisson_nll_loss  # noqa: F401
```

Comment:
should also add `poisson_nll_loss` in list of \_\_all\_\_ below in this file

Reply:
Done in new commit.